src/pages/guides/using/creating_rest_apis.md

```diff
@@ -153,6 +153,145 @@ curl -i -X OPTIONS https://adobeioruntime.net/...
 
 ## Securing the API endpoints
 
+### Oauth (using the Adobe Identity Management System)
+
+You can configure an action to require IMS validation for incoming requests using the following command: 
+```bash
+wsk action update <action_name> --web true -a require-validation true
+```
+
+Once IMS authentication has been enabled for an action, the only way to allow access to the action is by specifying a list of IMS scopes or client IDs that are permitted to invoke the action. 
+
+The following code snippet demonstrates how to configure access using a standard Swagger file and the `security` object:
+
+```json
+{
+    "basePath": "/v2",
+    "paths": {
+      "/ims-secure-endpoint": {
+        "get": {
+          "operationId": "your-namespaces/default/my-ims-secure-web-action.json",        
+          "security": [
+            {
+              "scopes_auth": {
+                "scopes" : [
+                  "write:pets",
+                  "read:pets"
+                ]
+              }
+            }
+          ]
+        }
+      }
+    },
+    "securityDefinitions": {
+        "scopes_auth": {
+          "type": "oauth2",
+          "authorizationUrl": "https://ims-na1-stg1.adobelogin.com/ims/validate_token/v1?client_id=ioruntime&type=access_token",
+          "flow": "implicit",
+          "scopes": {
+            "write:pets": "modify pets in your account",
+            "read:pets": "read your pets"
+          }
+        }
+    }
+}
+```
+This enables scope validation for the API endpoint, allowing requests with access tokens that have the scopes `write:pets` OR `read:pets`.
```

Comment:
Can we have a list of fields described explicitly? And which options are supported for each of them?
What is `flow`? Does it support only `implicit` or something else?
Are there any requirements for scopes definition (e.g., which characters are supported/prohibited)?

Reply:
This is standard swagger spec:  https://swagger.io/docs/specification/authentication/oauth2/

Reply:
Maybe we should add link to it?